.storybook/updated-type.css

```
@@ -15,35 +15,35 @@
   src:
     local('Sana Sans LCG 05 VF'),
     local('SanaSansLCG05VF-Regular'),
-    url(/SanaSansLCG05-Variable.ttf) format('truetype');
+    url(./SanaSansLCG05-Variable.ttf) format('truetype');
 }
 
 @font-face {
   font-family: 'Roboto';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)